src/main/java/org/mockito/internal/junit/DefaultStubbingLookupListener.java

```diff
@@ -20,6 +20,7 @@
 class DefaultStubbingLookupListener implements StubbingLookupListener {
 
     Strictness currentStrictness;
+    boolean mismatchesReported;
```

Comment:
Fixed it. Indeed it looks better now because after all I needed only 2 out of 4 setter/getter methods. The public/package local footprint of the class is smaller now (will push soon).

Thank you for useful code review feedback. My code is easy for me to understand (most of the time :), but I want it to be clear for other engs. Keep it coming.